src/main/java/com/autilus/hw0406/task2/Car.java

```diff
@@ -0,0 +1,73 @@
+package com.autilus.hw0406.task2;
+
+import java.io.*;
```

Comment:
doesn't use auto import
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

Reply:
I understand)